### PR TITLE
Update deprecate v1beta1 roles and bindings

### PR DESCRIPTION
**Pull request by afrittoli**
# Changes

Resources in rbac.authorization.k8s.io/v1beta are deprecated
in k8s v1.17 and removed in v1.22, so update resources accordingly

Signed-off-by: Andrea Frittoli <andrea.frittoli@uk.ibm.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

/kind misc

### CODE REVIEW

File: prow/prow.yaml [hunk 1]
```diff
@@ -496,7 +496,7 @@ metadata:
   name: "deck"
 ---
 kind: RoleBinding
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v11
```
Comment (vdemeester) [inline prow/prow.yaml:499]: v11 ? isn't it a typo and should it be `v1` instead ? 😝

**Review by vdemeester — CHANGES_REQUESTED** [review]

File: prow/prow.yaml (hunk 1 above)
Comment (afrittoli) [reply prow/prow.yaml:499]: 🤣 yeah the current version was very old :P

Nice catch, thanks

**Comment by afrittoli** [comment]
image

😆

**Review by vdemeester — APPROVED** [review]
/lgtm

**Review by jerop — APPROVED** [review]
/woof

**Pull request merged**